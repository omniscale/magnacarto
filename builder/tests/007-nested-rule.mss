#class {
    #class1{
        line-width: 13;
    }
}

#class {
    #class1{
        line-width: 13;
    }
    #class1{
        line-width: 13;
    }
}

#class {
    [filter='foo'] {
        line-width: 13;
    }
    line-width: 13;
    #class1{
        line-width: 13;
    }
    line-width: 13
}
