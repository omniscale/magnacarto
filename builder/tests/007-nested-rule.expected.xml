<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="class" filter-mode="first">
    <Rule>
      <Filter>([filter] = &#39;foo&#39;)</Filter>
      <LineSymbolizer stroke-width="13"></LineSymbolizer>
    </Rule>
    <Rule>
      <LineSymbolizer stroke-width="13"></LineSymbolizer>
    </Rule>
  </Style>
  <Layer name="class" srs="" status="off">
    <StyleName>class</StyleName>
  </Layer>
</Map>